torch/nn/utils/_stateless.py

```diff
@@ -0,0 +1,56 @@
+import contextlib
+
+import torch
+
+
+def _get_module_parameters_and_buffers(module):
```

Comment:
Do we want to remove this since it is not used?

Reply:
Mmm, it is not used in the example but I think we discussed that we should provide a function for users to get the state of the module, (parameters & weights without other object attributes).

Reply:
I feel like this is a stretch for now and we can add that later if users need it?